src/goSymbol.ts

```diff
@@ -51,27 +51,37 @@ export class GoWorkspaceSymbolProvider implements vscode.WorkspaceSymbolProvider
 		if (vscode.window.activeTextEditor && vscode.workspace.getWorkspaceFolder(vscode.window.activeTextEditor.document.uri)) {
 			root = vscode.workspace.getWorkspaceFolder(vscode.window.activeTextEditor.document.uri).uri.fsPath;
 		}
-		if (!root) {
+
+		let goConfig = vscode.workspace.getConfiguration('go', vscode.window.activeTextEditor ? vscode.window.activeTextEditor.document.uri : null);
+
+		if (!root && !goConfig.gotoSymbol.includeGoroot) {
 			vscode.window.showInformationMessage('No workspace is open to find symbols.');
 			return;
 		}
 
-		return getWorkspaceSymbols(root, query, token).then(results => {
-			let symbols: vscode.SymbolInformation[] = [];
-			convertToCodeSymbols(results, symbols);
-			return symbols;
-		});
+		let workspaceSymbols = getSymbols(root, query, token, goConfig);
+		let gorootSymbols = goConfig.gotoSymbol.includeGoroot && process.env.GOROOT
```

Comment:
`process.env.GOROOT` will be undefined unless the user has set the env var or added the `go.goroot` setting.
On my Mac, its neither.
We need to run `go env GOROOT` to get the goroot.
See https://github.com/Microsoft/vscode-go/blob/0.6.78/src/goInstallTools.ts#L298 for reference